File: Java/(Beecrowd)1036 - Bhaskara's Formula.java
import java.util.Scanner;

public class Main{
    
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        
        double a = input.nextDouble(); double b = input.nextDouble(); double c = input.nextDouble();
        
        if (a == 0){
            System.out.println("Impossivel calcular");
        } else{
            double d = (b*b) - 4 * a * c;
            if (d < 0){
                System.out.println("Impossivel calcular");
            } else{
                double x1 = (-b + Math.sqrt(d)) / (2 * a);
                double x2 = (-b - Math.sqrt(d)) / (2 * a);
                System.out.println(String.format("R1 = %.5f", x1));
                System.out.println(String.format("R2 = %.5f", x2));
            }
        }
    }
    
}

File: C/(Beecrowd)1036 - Bhaskara's Formula.c
#include <stdio.h>
#include <math.h>

int main() {
    double a, b, c, d;

    scanf("%lf %lf %lf", &a, &b, &c);
    d = b*b - 4*a*c;

    if (a <= 0 || d < 0) {
        printf("Impossivel calcular\n");
    } else {
        printf("R1 = %.5lf\n", (-b + sqrt(d))/(2*a));
        printf("R2 = %.5lf\n", (-b - sqrt(d))/(2*a));
    }

    return 0;
}